kano_profile/tracker/tracking_events.py

```diff
@@ -15,10 +15,33 @@
 
 
 def _hw_info():
+    # TODO: This import is done here to avoid potential
+    #       import loops with kano-peripherals.
+    from kano_peripherals.wrappers.detection import is_ck2_lite, is_ck2_pro, \
+        get_ck2_lite_version, get_ck2_pro_version
+
+    # If CKL/CKCs cannot be detected, defaults to Computer Kits without version.
+    kit_type = 'ck'
+    kit_version = None
+
+    # Detect CK Lite via the PiHat board.
+    if is_ck2_lite():
+        kit_type = 'ckl'
+        kit_version = get_ck2_lite_version()
+
+    # Detect CK Complete via the PowerHat board and other peripherals.
+    elif is_ck2_pro():
+        kit_type = 'ckc'
+        kit_version = get_ck2_pro_version()
+
     track_data('hw-info', {
         'keyboard': 'kano' if detect_kano_keyboard() else 'generic',
         'model': get_rpi_model(),
-        'partitions': get_partition_info()
+        'partitions': get_partition_info(),
+        'kit': {
+            'type': kit_type,
+            'version': kit_version.vstring
```

Comment:
Isn't this going to fail for the `kit_version = None` case?

Reply:
Yes, of course it will..... :man_facepalming:

Reply:
Fixed.